### DISCUSSION TITLE
Dev Setup

### DISCUSSION THREAD

**Discussion by harshalvk** [sigle/sigle#1022] [General]
Can any one, please tell me how do I setup the application locally????

**Comment by pradel** [comment]
Hey @Harshalvk to setup the application locally please have a look at the contributing guide https://github.com/sigle/sigle/blob/main/CONTRIBUTING.md. Let me know if you have issues, feel free to join discord too if you need some help.

**Reply by harshalvk** [reply]
Hey @pradel,
![image](https://github.com/sigle/sigle/assets/80589780/1ffd18bd-72c6-4bf5-81d0-a6002db4b729)
I got this issue

**Reply by pradel** [reply]
@Harshalvk are you up to date with main? There is no container postgres defined in the docker compose file

**Reply by harshalvk** [reply]
I have just cloned the **repo**

**Reply by harshalvk** [reply]
![image](https://github.com/sigle/sigle/assets/80589780/74eea9a1-11f1-4c2b-99cb-c451ee5928a3)
[Check this out](https://github.com/sigle/sigle/blob/main/docker-compose.yaml)

**Reply by pradel** [reply]
Ohhhh that's my bad, the correct command is `docker compose up`

**Reply by harshalvk** [reply]
Thank you Sir.
